doc/user-guide-v1.adoc

```diff
@@ -430,8 +437,11 @@ And here is the set of descriptions corresponding to each configurable field.
 | `certManagerCertDuration`   | The cert-manager issued service certificate's duration before expiry in link:++https://pkg.go.dev/time#ParseDuration++[Go time.Duration] string format. The default value is 2160h (90 days). To learn more about this field see link:#generating-certificates-with-certificate-manager[Generating certificates with certificate manager].
 | `defaultHostname`   | The default hostname for the OpenLibertyApplication Route or Ingress URL when .spec.expose is set to true. To learn more about this field see link:#expose-applications-externally[Expose applications externally (`.spec.expose`, `.spec.createKnativeService`, `.spec.route`)].
 | `operatorLogLevel` | The log level for the Liberty operator. The default value is `info`, other options are `warning`, `fine`, `finer`, `finest`. The log level can be dynamically modified and takes effect immediately.
-| `reconcileIntervalMinimum` | The default value of the base reconciliation interval in seconds is _5_. The operator runs the reconciliation loop every reconciliation interval seconds for each instance. If an instance's status conditions remain unchanged for 2 consecutive reconciliation loops, the reconciliation interval increases to reduce the reconciliation frequency. The interval increases based on the base reconciliation interval and specified increase percentage. For more information on the operator's reconciliation frequency, see link:#viewing-reconciliation-frequency-in-the-status[Viewing reconciliation frequency in the status].
-| `reconcileIntervalIncreasePercentage` | When the reconciliation interval increases, the increase is calculated as a specified percentage of the current interval. To disable the reconciliation interval increase, set the value to _0_.
+| `reconcileIntervalMinimum` | The default value of the minimum reconciliation interval in seconds is _5_. The operator runs the reconciliation loop every reconciliation interval seconds for each instance. If an instance's status conditions remain unchanged, the reconciliation interval increases to reduce the reconciliation frequency. The interval increases based on the base reconciliation interval and specified increase percentage. For more information on the operator's reconciliation frequency, see link:#viewing-reconciliation-frequency-in-the-status[Viewing reconciliation frequency in the status].
```

Comment:
Noticed a difference between the description of `reconcileIntervalMinimum` in OLO and WLO:

WLO has the portion in bold while OLO doesn't:
_If an instance's status conditions remain unchanged **for 2 consecutive reconciliation loops**, the reconciliation interval increases to reduce the reconciliation frequency._

Reply:
@halim-lee Thank you for opening the workitem to update WLO docs to match this OLO update.